src/VisualStudio/Core/Def/Implementation/ChangeSignature/ChangeSignatureDialogViewModel.cs

```diff
@@ -356,10 +371,66 @@ public bool IsOkButtonEnabled
 
                 _selectedIndex = newSelectedIndex;
 
-                NotifyPropertyChanged("CanMoveUp");
-                NotifyPropertyChanged("CanMoveDown");
-                NotifyPropertyChanged("CanRemove");
-                NotifyPropertyChanged("CanRestore");
+                NotifyPropertyChanged(nameof(CanMoveUp));
+                NotifyPropertyChanged(nameof(MoveUpAutomationText));
+                NotifyPropertyChanged(nameof(CanMoveDown));
+                NotifyPropertyChanged(nameof(MoveDownAutomationText));
+                NotifyPropertyChanged(nameof(CanRemove));
+                NotifyPropertyChanged(nameof(RemoveAutomationText));
+                NotifyPropertyChanged(nameof(CanRestore));
+                NotifyPropertyChanged(nameof(RestoreAutomationText));
+            }
+        }
+
+        public string MoveUpAutomationText
+        {
+            get
+            {
+                if (!CanMoveUp)
+                {
+                    return string.Empty;
```

Comment:
The convention is that if an element is disabled, the screen reader doesn't read it? (Asking for edification, not because I disagree)